torch/onnx/symbolic_opset12.py

```diff
@@ -92,3 +93,59 @@ def ge(g, input, other):
 
 def le(g, input, other):
     return g.op('LessOrEqual', input, other)
+
+@parse_args('v', 'i', 'v', 'v')
+def unfold(g, input, dimension, size, step):
```

Comment:
Is this function for dynamic input only? If it is static input and opset-12, should we fall back to opset-9 implementation?

Reply:
looks like only `size` and `step` may be dynamic. Maybe that can be used as a differentiation as to whether this need to be exported dynamically. 
Also I wonder why `sizedim` is statically used, given that its only usage is as input to range op. In other words, looks like we can support dynamic `sizedim` as well if static value is not inferred.

Reply:
sizedim is not the input, dimension is. Not sure if we can support dynamic input of dimension.